### ISSUE TITLE
Upgrade to latest version of blacklight_range_limit

### ISSUE THREAD

**Issue by bess** [pulibrary/pdc_discovery#703]
We're going to need to upgrade to BL 8 at some point, but right now we can't because blacklight_range_limit (which we're using) won't work with BL 8 yet. 

Test whether the WIP on upgrading it will work with this app: https://github.com/projectblacklight/blacklight_range_limit/pull/266

Blocked until there is documentation and an alpha release.

**Comment by bess** [comment]
Now unblocked and they authors would really like our feedback before they do an official release.

**Comment by kelynch** [comment]
There is now an alpha release.

**Comment by kelynch** [comment]
Closing in favor of https://github.com/pulibrary/pdc_discovery/issues/959
